test/integration/test_http.py

```diff
@@ -465,6 +465,31 @@ def test_query(self):
         status = txn.end()
         self.assertTrue(status.is2xxOK())
 
+    def test_txn_timeout(self):
+        # Begin Txn with timeout 1s
```

Comment:
could we make these timeouts tighter so we don't have to wait so much for the entire test to complete?